actionpack/lib/action_dispatch/http/content_security_policy.rb

```diff
@@ -227,8 +240,7 @@ def plugin_types(*types)
       end
     end
 
-    # Enable the [report-uri]
-    # (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri)
+    # Enable the [report-uri](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri)
```

Comment:
I don't think this is necessary, the link seems to be working fine:
https://edgeapi.rubyonrails.org/classes/ActionDispatch/ContentSecurityPolicy.html#method-i-report_uri